src/bot.py

```diff
@@ -75,40 +123,43 @@ def tweet_loop(self, api):
                 else:
                     api.update_status(self.generate_batch())
                     logging.info("Tweeted out a new flower bed!")
-                    logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
+                    logging.info("The next tweet is scheduled to be made in {} seconds".format(cooldown))
                     time.sleep(cooldown)
-            except:
-                print(sys.exc_info())
-                logging.critical("Tweeting failed due to ratelimit. Waiting {} more minutes.".format(cooldown))
-                time.sleep(cooldown)
+                    raise tweepy.TweepError("error")
+            except Exception as ex:
+                exception = type(ex).__name__
                 
+                if exception == 'RateLimitError':
+                    logging.critical("Tweeting failed due to ratelimit. Waiting {} more seconds.".format(cooldown))
+                    time.sleep(cooldown)

```

Comment:
Could you keep the exception handlers in the same place as the other exception handlers? (Bottom of the script)

Reply:
suggestion taken.